File: app/src/main/java/com/crm/crmapp/CRMDashboardModel/Result2.java

package com.crm.crmapp.CRMDashboardModel;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Result2 {

    @SerializedName("cnt")
    @Expose
    private Integer cnt;
    @SerializedName("amt")
    @Expose
    private Integer amt;
    @SerializedName("exp_date")
    @Expose
    private String expDate;

    public Integer getCnt() {
        return cnt;
    }

    public void setCnt(Integer cnt) {
        this.cnt = cnt;
    }

    public Integer getAmt() {
        return amt;
    }

    public void setAmt(Integer amt) {
        this.amt = amt;
    }

    public String getExpDate() {
        return expDate;
    }

    public void setExpDate(String expDate) {
        this.expDate = expDate;
    }

}

File: app/src/main/java/com/crm/crmapp/expense/model/Result2.kt
package com.crm.crmapp.expense.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName



class Result2 {

    @SerializedName("cnt")
    @Expose
    private var cnt: Int? = null
    @SerializedName("amt")
    @Expose
    private var amt: Double? = null
    @SerializedName("exp_date")
    @Expose
    private var expDate: String? = null

    fun getCnt(): Int? {
        return cnt
    }

    fun setCnt(cnt: Int?) {
        this.cnt = cnt
    }

    fun getAmt(): Double? {
        return amt
    }

    fun setAmt(amt: Double?) {
        this.amt = amt
    }

    fun getExpDate(): String? {
        return expDate
    }

    fun setExpDate(expDate: String) {
        this.expDate = expDate
    }
}